.circleci/config.yml

```diff
@@ -35,6 +26,12 @@ jobs:
       - run:
           name: Run golangci-lint
           command: "! $(go env GOPATH)/bin/golangci-lint run ./... 2>&1 | read"
```

Comment:
Should we change this to use `test -z` too? If `go fmt` was not working, `golangci-lint` most likely wasn't either. Thanks!

Reply:
Sure enough, it was definitely not working, although `test -z` is the wrong answer in this case. Also, turns out golangci-lint has a thing to check formatting, so we no longer need the other separate step.